docs/management_commands.rst

```diff
@@ -0,0 +1,74 @@
+Django Management Commands
+==========================
+
+Export Command
+--------------
+
+The ``export`` command allows you to export data from a specified Django model
+or a resource class. The exported data can be saved in different formats, such
+as CSV or XLSX.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py export <format> <resource> [--encoding ENCODING]
+
+- **format**: Specify the format in which the data should be exported. -
+- **resource**: Specify the resource or model to export. Accepts a resource class or a model class in dotted path format. - **--encoding** (optional): Specify the encoding (e.g., 'utf-8') to be used for the exported data.
+
+Example
+-------
+
+.. code-block:: bash
+
+    python manage.py export CSV auth.User
+
+This command will export the User model data in CSV format using utf-8
+encoding.
+
+Another example:
+
+.. code-block:: bash
+
+    python manage.py export XLSX mymodule.resources.MyResource
+
+This command will export the data from ``MyResource`` resource in XLSX format
+using utf-8 encoding.
```

Comment:
I think this comment is incorrect - an XLSX binary export will be in the native binary type not utf-8.